### PR TITLE
JBF-6

### PR DESCRIPTION
**Pull request by Sialenius**

### CODE REVIEW

File: src/model/CustomArrayList.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            } else {
+            for (int i = index; i < filled - 1; i++) {
+                list[i] = list[i+1];
+            }
+            filled--;
+        }
+    }
+
+
+    public boolean equals(CustomArrayList<T> secondList) {
```
Comment (AtticSiol) [inline src/model/CustomArrayList.java]: try to add @Override explicitly

File: src/control/TicketService.java [hunk 2]
```diff
@@ -47,9 +42,40 @@ public static void main(String[] args)  {
...
+
+    numbers.put(1);
+    numbers.put(2);
+    numbers.put(3);
+    numbers.put(4);
+    numbers.put(5);
+    System.out.println(numbers);
+
+
+    CustomHashSet<String> strings = new CustomHashSet<>(2);
```
Comment (AtticSiol) [inline src/control/TicketService.java]: name collection variables as `stringsSet` for example

Comment (AtticSiol) [inline src/control/TicketService.java]: add example how to use your iterator method

File: src/model/CustomArrayList.java (hunk 1 above)
Comment (Sialenius) [reply src/model/CustomArrayList.java]: Fixed. I changed the method's input argument and added the @Override explicity

File: src/control/TicketService.java (hunk 2 above)
Comment (Sialenius) [reply src/control/TicketService.java]: Fixed. I renamed variables' names

Comment (Sialenius) [reply src/control/TicketService.java]: Done. I added the example how to use the custom iterator

**Review by AtticSiol — APPROVED** [review]

**Pull request merged**